modules/appeals_api/app/swagger/v1/decision_reviews.yaml

```diff
@@ -256,9 +255,6 @@ paths:
                       a veteran's contestable issues to select from, and include
                       the _ContestableIssue_ objects that you want to contest in this
                       array.
-                      When including a _ContestableIssue_ obtained from the
-                      `/contestable_issues` endpoint, use the `notes` field to provide
-                      additional information about contesting this issue.
```

Comment:
this change is just to be in-sync with Caseflow's documentation (which accidentally became more up-to-date)